qcodes/instrument_drivers/AlazarTech/ATS.py

```diff
@@ -33,185 +32,38 @@ class AlazarTech_ATS(Instrument):
-        568: 'ApiBadPinDirection',
-        569: 'ApiPciTimeout',
-        570: 'ApiDmaChannelClosed',
-        571: 'ApiDmaChannelError',
-        572: 'ApiInvalidHandle',
-        573: 'ApiBufferNotReady',
-        574: 'ApiInvalidData',
-        575: 'ApiDoNothing',
-        576: 'ApiDmaSglBuildFailed',
-        577: 'ApiPMNotSupported',
-        578: 'ApiInvalidDriverVersion',
-        579: ('ApiWaitTimeout: operation did not finish during '
-              'timeout interval. Check your trigger.'),
-        580: 'ApiWaitCanceled',
-        581: 'ApiBufferTooSmall',
-        582: ('ApiBufferOverflow:rate of acquiring data > rate of '
-              'transferring data to local memory. Try reducing sample rate, '
-              'reducing number of enabled channels, increasing size of each '
-              'DMA buffer or increase number of DMA buffers.'),
-        583: 'ApiInvalidBuffer',
-        584: 'ApiInvalidRecordsPerBuffer',
-        585: ('ApiDmaPending:Async I/O operation was successfully started, '
-              'it will be completed when sufficient trigger events are '
-              'supplied to fill the buffer.'),
-        586: ('ApiLockAndProbePagesFailed:Driver or operating system was '
-              'unable to prepare the specified buffer for DMA transfer. '
-              'Try reducing buffer size or total number of buffers.'),
-        587: 'ApiWaitAbandoned',
-        588: 'ApiWaitFailed',
-        589: ('ApiTransferComplete:This buffer is last in the current '
-              'acquisition.'),
-        590: 'ApiPllNotLocked:hardware error, contact AlazarTech',
-        591: ('ApiNotSupportedInDualChannelMode:Requested number of samples '
-              'per channel is too large to fit in on-board memory. Try '
-              'reducing number of samples per channel, or switch to '
-              'single channel mode.')
-    }
-
-    _board_names = {
-        0: 'ATS_NONE',
-        1: 'ATS850',
-        2: 'ATS310',
-        3: 'ATS330',
-        4: 'ATS855',
-        5: 'ATS315',
-        6: 'ATS335',
-        7: 'ATS460',
-        8: 'ATS860',
-        9: 'ATS660',
-        10: 'ATS665',
-        11: 'ATS9462',
-        12: 'ATS9434',
-        13: 'ATS9870',
-        14: 'ATS9350',
-        15: 'ATS9325',
-        16: 'ATS9440',
-        17: 'ATS9410',
-        18: 'ATS9351',
-        19: 'ATS9310',
-        20: 'ATS9461',
-        21: 'ATS9850',
-        22: 'ATS9625',
-        23: 'ATG6500',
-        24: 'ATS9626',
-        25: 'ATS9360',
-        26: 'AXI9870',
-        27: 'ATS9370',
-        28: 'ATU7825',
-        29: 'ATS9373',
-        30: 'ATS9416',
-        31: 'ATS9637',
-        32: 'ATS9120',
-        33: 'ATS9371',
-        34: 'ATS9130',
-        35: 'ATS9352',
-        36: 'ATS9453',
-    }
-
     @classmethod
-    def find_boards(cls, dll_path: str=None) -> List[dict]:
-        """
-        Find Alazar boards connected
+    def find_boards(cls, dll_path: str = None) -> List[dict]:
```

Comment:
should we not preserve the doc string here in an updated version?

Reply:
sure!